.github/workflows/update-copyright-notices.yml

```
@@ -0,0 +1,39 @@
+name: Update copyright notices
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    types:
+      - closed
+
+jobs:
+  check-and-update-copyright-notices:
+    if: ${{ github.event_name == 'push' }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - run: ./tools/update-copyright-notices.sh
+      - run: |
+          git config --global user.email "actions@github.com"
+          git config --global user.name "github-actions[bot]"
+          pr_branch=update-copyright-notices-$(date +%Y-%m-%d-%H-%M-%S)
+          git checkout -b ${pr_branch}
+          git add .
+          git commit -m "Update copyright notices for $(date +%Y)" || exit 0
```

```suggestion
          git commit -m "AUTO: Update copyright notices for $(date +%Y)" || exit 0
```

Similar to what the website does.

---

Done (although the fact that it's automated should already be clear by looking at the author).

---

Yes. Just a consistency thing.